src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -1047,9 +2390,9 @@ module Tick : S = struct
                  * cell (var (Witness 13, Curr)) ) ) )
       ; ( Index ChaCha0
         , lazy
-            (enabled_if
+            (if_feature
```

Comment:
Mmmm... after looking at the block of code starting on row 2122 I am starting to get the feeling that this is perhaps autogenerated? Is it? That looks almost like ASCII art :O

Reply:
I am not referring to the code that comes after the selected row, but 2122 (that I was not able to write comments about using the GUI)

Reply:
Okay, David told me this whole file is generated automatically 😅